checkov/terraform/checks/resource/alicloud/RDSRetention.py

```diff
@@ -0,0 +1,32 @@
+from checkov.common.models.enums import CheckCategories,CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class RDSRetention(BaseResourceCheck):
+    def __init__(self):
+
+        """
+        'sql_collector_status' should be defined and set to "Enabled"  (I know, really, who does that)
+         and 'sql_collector_config_value' should be defined and set to 180 or more
+        """
+        name = "Ensure RDS Instance SQL Retention Period should be greater than 180"
+        id = "CKV_ALI_25"
+        supported_resources = ['alicloud_db_instance']
+        categories = [CheckCategories.NETWORKING]
```

Comment:
```suggestion
        categories = [CheckCategories.LOGGING]
```
is it logs or is it something else?